File: openapi-generator/php-ze-ph/src/App/DTO/AllRouteJobUpdates.php
<?php
declare(strict_types=1);

namespace App\DTO;

use Articus\DataTransfer\Annotation as DTA;

/**
 */
class AllRouteJobUpdates
{
    /**
     * @DTA\Data(field="job_updates", nullable=true)
     * TODO check validator and strategy are correct and can handle container item type
     * @DTA\Strategy(name="ObjectArray", options={"type":\App\DTO\JobUpdateObject::class})
     * @DTA\Validator(name="Collection", options={"validators":{
     *     {"name":"Dictionary", "options":{"type":\App\DTO\JobUpdateObject::class}}
     * }})
     * @var \App\DTO\JobUpdateObject[]
     */
    public $job_updates;
    /**
     * Sequence ID of the last update returned in the response
     * @DTA\Data(field="sequence_id", nullable=true)
     * @DTA\Validator(name="Type", options={"type":"string"})
     * @var string
     */
    public $sequence_id;
}

File: openapi-generator/dart-jaguar/lib/model/all_route_job_updates.dart
import 'package:jaguar_serializer/jaguar_serializer.dart';

import 'package:openapi/model/job_update_object.dart';
part 'all_route_job_updates.jser.dart';

class AllRouteJobUpdates {
  
  @Alias('job_updates')
  final List<JobUpdateObject> jobUpdates;
   /* Sequence ID of the last update returned in the response */
  @Alias('sequence_id')
  final String sequenceId;
  

  AllRouteJobUpdates(
      

{
     this.jobUpdates = const [],  
     this.sequenceId = null 
    
    }
  );

  @override
  String toString() {
    return 'AllRouteJobUpdates[jobUpdates=$jobUpdates, sequenceId=$sequenceId, ]';
  }
}

@GenSerializer(nullableFields: true)
class AllRouteJobUpdatesSerializer extends Serializer<AllRouteJobUpdates> with _$AllRouteJobUpdatesSerializer {

}
